great-idea-website/css/index.css

```diff
@@ -100,21 +102,75 @@ header nav .logo {
 
 .cta1 {
 	display: relative;
-	width: 400px;
-	margin-top: 50px;
+	width: 300px;
+	margin-top: 70px;
 	margin-left: 50px;
-	font-size: 40pt;
+	font-size: 50pt;
+	/* border: 1px solid lightcoral; */
 }
 
 .cta2 {
 	display: relative;
-	margin-top: 50px;
+	width: 300px;
+	margin-top: 20px;
 	margin-left: 50px;
 	font-size: 26pt;
+	/* border: 1px solid lightcoral; */
 }
 
 .cta3 {
 	display: relative;
 	margin-top: -300px;
 	margin-left: 400px;
+}
```

Comment:
I think you wanted to use position: relative, display: relative it is not valid property value